src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.model;
+
+import java.util.Arrays;
+
+public class FruitTransaction {
+    private final Operation operation;
+    private final String fruit;
+    private final int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
```

Comment:
You can use public record here and don't need to create constructor and getters
```suggestion
public record FruitTransaction(Operation operation, String fruit, int quantity) {
```

Reply:
Done, thanks 👍